pegasus/sites.v3/code.org/public/educate/csp-draft.haml

```diff
@@ -0,0 +1,96 @@
+---
+title: Computer Science Principles
+theme: responsive_full_width
+---
+
+%link{href: '/css/generated/design-system-pegasus.css', rel: 'stylesheet'}
+%link{href: "/css/generated/csp-styles.css", rel: "stylesheet"}
+
+%section.hero-banner-basic.overlay
+  .wrapper.flex-container.justify-space-between.align-items-center.wrap
+    .text-wrapper
+      %h1{style: "color: white"}
+        =hoc_s(:curriculum_name_csp)
+      %p{style: "color: white"}
+        =hoc_s(:csp_page_top_desc)
+      %a.link-button{href: CDO.studio_url("/courses/csp-2023?viewAs=Instructor")}
+        =hoc_s(:call_to_action_explore_course)
+    %figure
+      %img{src: "/images/banners/banner-csp-phone-icon.png", alt: ""}
+    .clear
+
+%section
+  .wrapper
+    %div.col-50{style: "margin-bottom: 1rem;"}
+      =view :display_video_thumbnail, id: "intro_csp", video_code: "DMr1iFYacGQ", play_button: 'center', letterbox: 'false'
+    .text-wrapper.col-45{style: "float: right;"}
+      %h2=hoc_s(:csp_page_intro_heading)
+      %ul
+        %li=hoc_s(:csp_page_intro_list_01)
+        %li=hoc_s(:csp_page_intro_list_02)
+        %li=hoc_s(:csp_page_intro_list_03)
+        %li=hoc_s(:csp_page_intro_list_04)
+      %a.link-button{href: CDO.studio_url("/courses/csp-2023?viewAs=Instructor")}
+        =hoc_s(:call_to_action_explore_course)
+
+    .clear
+
+%section.bg-neutral-light
+  .wrapper
+    = view :curriculum_glance_section, grades: hoc_s(:module_grade_9_12), duration: hoc_s(:module_duration_school_year), level: hoc_s(:module_level_beginner), tools: hoc_s(:csp_page_curriculum_glance_tools), devices: hoc_s(:csp_page_curriculum_glance_devices), professional_learning: hoc_s(:csp_page_curriculum_glance_professional_learning), topics: hoc_s(:csp_page_curriculum_glance_topics), accessibility_features: hoc_s(:csp_page_curriculum_glance_professional_accessibility_features), languages: hoc_s(:csp_page_curriculum_glance_professional_languages)
+
+%section.quote.bg-neutral-dark
+  .wrapper
+    %div.quotes-wrapper
+      %p.quote-text.large
+        =hoc_s(:csp_page_quote)
+      %p.no-margin-bottom
+        Jennifer Douglass
+        %span •
+        =hoc_s(:teacher_label_csp)
+
+%section
+  .wrapper
+    %h2=hoc_s(:heading_engaging_projects)
+    = view :projects_csp_page
+
+%section.bg-neutral-light
+  .wrapper
+    = view :resources_tabs
+
+%section
+  .wrapper
+    %h2=hoc_s(:heading_professional_learning)
```

Comment:
I might put this in a partial when I start working on the other curriculum pages.